openapi.yml

```diff
@@ -4474,6 +4481,9 @@ components:
                     type: string
                 email:
                     type: string
+                registration_id:
+                    description: UUID of a Registration
```

Comment:
wondering if we should add descriptions to the now optional fields that they are in fact ignored if a registration_id is present